src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,97 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import tasks.Task;
+
+/**
+ * Handles interaction with the user and prints information about the program.
+ */
+public class Ui {
+    /**
+     * The message to be shown before quitting the application.
+     * @return The goodbye message.
+     */
+    public String getGoodbyeMessage() {
+        return "Bye. Hope to see you again soon!";
+    }
+
+    /**
+     * Human readable information about the task list after adding a task.

```

Comment:
Do you think it would be better to start the first sentence with a verb such as returns?